content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -252,12 +488,23 @@ certificate. To enable this feature pass the following flag to the kubelet:
 
 {{< note >}}
 The CSR approving controllers implemented in core Kubernetes do not
-approve node serving certificates for [security
+approve node _serving_ certificates for [security
 reasons](https://github.com/kubernetes/community/pull/1982). To use
 `RotateKubeletServerCertificate` operators need to run a custom approving
 controller, or manually approve the serving certificate requests.
 {{< /note >}}
 
+## kube-proxy
```

Comment:
I would not include kube-proxy in this page.

Reply:
A few thoughts:
* kube-proxy is one of many things that can run on nodes... calling it out specifically here makes it seem more special than it is.
* I don't think we should advocate pointing kube-proxy at kubelet credentials on disk. cc @mikedanese, xref https://github.com/kubernetes/kubernetes/issues/70958 https://github.com/kubernetes/kubernetes/issues/62747

Reply:
> kube-proxy is one of many things that can run on nodes... calling it out specifically here makes it seem more special than it is.

If I look at the docs for installing kube, at a minimum, each node will have:

* container runtime (historically docker, although that can be anything with CRI, but it must exist). Either way, this does not have to authenticate to the master plane, since it is controlled by kubelet
* kubelet - covered by this doc
* kube-proxy - subject of this discussion

(do you know how many times I had to edit this because of the damn MacBook Pro keyboard? What really bad hardware engineering...)

kube-proxy is required to do things correctly, it is part of the core control plane, and the docs tell me, "install container runtime, kubelet, kube-proxy." 

Doesn't that make it special? What else might I run on a node that would make it a worker and is part of the control plane?

Reply:
BTW, not trying to make a point "just because"; this really is how many of us users see it.

Reply:
Would a network plugin be part of the official control plane?

> If we want a section that describes approaches for running other things on the node, and kube-proxy is given as an example, that seems more reasonable.

Yeah, that works. Just mildly different from this one anyways. :-)

> The two approaches I would highlight would be to manually provision and provide credentials, or to run as a daemonset.

In other words, get rid of the "shared credential" approach?

Let's move this all to the other open PR.